app/styles/_request_purpose.scss

```diff
@@ -322,3 +347,25 @@
   margin-top: -2%;
 }
 
+.select-error-message {
+  display: none;
+  background: #ED4C4E;
```

Comment:
It would be good to have color variables for the colors used at multiple places for common use-case like highlighting invalid inputs. (used at two other places in the app https://github.com/crossroads/stock.goodcity/search?q=ED4C4E&unscoped_q=ED4C4E)

Define variables here https://github.com/crossroads/stock.goodcity/blob/master/app/styles/_settings.scss

Reply:
Good suggestion @swatijadhav :+1: